bin/run-in-docker.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
This file is missing the executable bit

Reply:
It's probably me, but I couldn't get this file working under Windows Subsystem for Linux 2:

```
Copying exercise files into /output
cp: cannot create regular file '/output/README.md': Permission denied
cp: cannot create regular file '/output/elm-package.json': Permission denied
cp: cannot create directory '/output/src': Permission denied
cp: cannot create directory '/output/tests': Permission denied
```

I was calling it with: `./bin/run-in-docker.sh space-age /home/erik/exercism/elm/space-age /tmp/space-age-output`

Reply:
What do you mean by "This file is missing the executable bit"

Reply:
I can run this without errors on git bash 
```
./bin/run-in-docker.sh space-age /C/Users/cburge/Documents/repos/space-age/ /C/Users/cburge/Documents/repos/space-age-output/
```

It builds the docker image, but the `docker run ..` statement doesn't do anything.

I see the same problem if I just run the docker statement directly in Git Bash, and there is nothing in `docker logs ...`
```
docker run --network none --read-only --mount type=bind,src=/C/Users/cburge/Documents/repos/space-age/,dst=/solution --mount type=bind,src=/C/Users/cburge/Documents/repos/space-age-output/,dst=/output elm-representer $SLUG /solution /output
```

The same command does work in powershell, and docker is docker, so I think its just a linux on windows thing. I propose to ignore it if that's ok?

I think it would work (or at least be more easily debuggable), for a Mac or linux user

Reply:
I think this is a git bash peculiarity, see:

```
erik@VELPIE:~/elm-representer$ ls -al ./bin/
total 20
drwxr-xr-x 2 erik erik 4096 Dec  4 09:42 .
drwxr-xr-x 5 erik erik 4096 Dec  4 09:42 ..
-rw-r--r-- 1 erik erik 1354 Dec  4 09:42 run-in-docker.ps1
-rw-r--r-- 1 erik erik 1348 Dec  4 09:42 run-in-docker.sh
-rw-r--r-- 1 erik erik 1028 Dec  4 09:42 run.sh
```

> The same command does work in powershell, and docker is docker, so I think its just a linux on windows thing. I propose to ignore it if that's ok?

Sure. Maybe create an issue to keep track of that?

Reply:
I've added #3 for this.